Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -72,6 +72,11 @@ elif [ ${RELEASE_CODENAME} == 'focal' ]; then
     SITLCFML_VERSION="2.5"
     PYTHON_V="python3"
     PIP=pip3
+elif [ ${RELEASE_CODENAME} == 'groovy' ]; then
```

Comment:
Could we group this with focal ?

Reply:
I'm still considering that.  I'm leaning towards "not" ATM - until groovy is released these things could change.  Even post-that I'm not sure its worth doing as we're already seeing a pattern of a per-release if/else thing.  I may have prefered the "auto-sensing" code there, but simple is probably better.